### PR TITLE
Adapt to the new grub-mender-grubenv-* tools.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
Being tested in https://github.com/mendersoftware/meta-mender/pull/1390.

File: tests/test_update.py [hunk 1]
```diff
@@ -192,13 +193,15 @@ def test_network_based_image_update(
             s3_address,
         )
 
-        output = connection.run("fw_printenv bootcount").stdout
+        bootenv_print, bootenv_set = bootenv_tools(connection)
```
Comment (lluiscampos) [inline tests/test_update.py:196]: `bootenv_set` here is unused
```suggestion
        bootenv_print, _ = bootenv_tools(connection)
```

**Review by lluiscampos — APPROVED** [review]
Nice solution

File: tests/test_update.py (hunk 1 above)
Comment (kacf) [reply tests/test_update.py]: Thanks, fixed!

**Review by lluiscampos — APPROVED** [review]

File: tests/test_update.py (hunk 1 above)
Comment (lluiscampos) [reply tests/test_update.py:193]: Somehow you re-introduced this in your last force-push.

**Comment by lluiscampos** [comment]
I am not marking for rework because it is a really minor thing, but see my comment above.

**Review by lluiscampos — APPROVED** [review]

File: tests/test_update.py (hunk 1 above)
Comment (kacf) [reply tests/test_update.py:193]: Wah, how did that happen? Did I use an old branch? I will have to compare with the SHAs in this PR now, I don't trust that I didn't mess up something else too.

Comment (kacf) [reply tests/test_update.py]: This should be fixed now, and I verified to be the best of my ability that I didn't miss anything else in the force push.

**Pull request merged**